### PR TITLE
fix: window picker can't be correctly rendered when cmdheight = 0

### LINKED ISSUE #1361: Picker window doesn't work as should with `cmdheight` set to 0
### Description

With `cmdheight` set to 0, the picker window doesn't work as expected. The letters to choose the buffer are not displayed.

### Neovim version

```text
NVIM v0.8.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wsuggest-attribute=pure -Wsuggest-attribute=const -Wsuggest-attribute=malloc -Wsuggest-attribute=cold -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/data/Development/github.com/neovim/neovim/build/config -I/data/Development/github.com/neovim/neovim/src -I/data/Development/github.com/neovim/neovim/.deps/usr/include -I/usr/include -I/data/Development/github.com/neovim/neovim/build/src/nvim/auto -I/data/Development/github.com/neovim/neovim/build/include
Compiled by root@neutron.node.inet

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info
```

### Operating system and version

Linux 5.15.0-39-generic #42-Ubuntu SMP Thu Jun 9 23:42:32 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

### nvim-tree version

f262236

### Minimal config

Global config

```
set cmdheight=0
```

nvim-tree config:

```lua
t.setup {
  filesystem_watchers = {
    enable = true,
  },
  auto_reload_on_write = true,
  disable_netrw = true,
  sort_by = 'name',
  renderer = {
    add_trailing = true,
    indent_markers = {
      enable = true,
      icons = {
        corner = icons.uiCorner,
        edge = icons.uiEdge,
        none = icons.uiNone,
      },
    },
    icons = {
      show = {
        git = true,
        folder = true,
        file = true,
        folder_arrow = false,
      },
      webdev_colors = true,
      glyphs = {
        default = icons.file,
        symlink = icons.symlink,
        git = {
          unstaged = icons.gitUnstaged,
          staged = icons.gitStaged,
          unmerged = icons.gitUnmerged,
          renamed = icons.gitRenamed,
          untracked = icons.gitUntracked,
          deleted = icons.gitDeleted,
          ignored = icons.gitIgnored,
        },
        folder = {
          arrow_open = icons.arrowOpen,
          arrow_closed = icons.arrowClosed,
          default = icons.folder,
          open = icons.folderOpen,
          empty = icons.folderNoBg,
          empty_open = icons.folderOpenNoBg,
          symlink = icons.folderSymlink,
          symlink_open = icons.folderSymlink,
        },
      },
    },
  },
  hijack_netrw = true,
  open_on_setup = false,
  ignore_ft_on_setup = {},
  open_on_tab = false,
  hijack_cursor = false,
  update_cwd = true,
  diagnostics = {
    enable = false,
    icons = {
      hint = icons.warningCircle,
      info = icons.info,
      warning = icons.warningTriangle,
      error = icons.error,
    },
  },
  update_focused_file = {
    enable = true,
    update_cwd = true,
    ignore_list = {},
  },
  system_open = {
    cmd = nil,
    args = {},
  },
  filters = {
    dotfiles = true,
    custom = {
      '^.git$',
      '^node_modules',
    },
  },
  git = {
    enable = true,
    ignore = false,
    timeout = 500,
  },
  view = {
    width = 35,
    height = 30,
    hide_root_folder = true,
    side = 'left',
    adaptive_size = true,
    mappings = {
      custom_only = false,
      list = {
        { key = { '<CR>' }, cb = tree_cb('edit') },
        { key = { '+' }, cb = tree_cb('cd') },
        { key = 'v', cb = tree_cb('vsplit') },
        { key = 'h', cb = tree_cb('split') },
        { key = 't', cb = tree_cb('tabnew') },
        { key = '<', cb = tree_cb('prev_sibling') },
        { key = '>', cb = tree_cb('next_sibling') },
        { key = 'P', cb = tree_cb('parent_node') },
        { key = '<BS>', cb = tree_cb('close_node') },
        { key = '<S-CR>', cb = tree_cb('close_node') },
        { key = '<Tab>', cb = tree_cb('preview') },
        { key = 'K', cb = tree_cb('first_sibling') },
        { key = 'J', cb = tree_cb('last_sibling') },
        { key = 'I', cb = tree_cb('toggle_ignored') },
        { key = 'H', cb = tree_cb('toggle_dotfiles') },
        { key = 'R', cb = tree_cb('refresh') },
        { key = 'a', cb = tree_cb('create') },
        { key = 'd', cb = tree_cb('remove') },
        { key = 'r', cb = tree_cb('rename') },
        { key = '<C-r>', cb = tree_cb('full_rename') },
        { key = 'x', cb = tree_cb('cut') },
        { key = 'c', cb = tree_cb('copy') },
        { key = 'p', cb = tree_cb('paste') },
        { key = 'y', cb = tree_cb('copy_name') },
        { key = 'Y', cb = tree_cb('copy_path') },
        { key = 'gy', cb = tree_cb('copy_absolute_path') },
        { key = '<c', cb = tree_cb('prev_git_item') },
        { key = '>c', cb = tree_cb('next_git_item') },
        { key = '-', cb = tree_cb('dir_up') },
        { key = 'q', cb = tree_cb('close') },
        { key = '?', cb = tree_cb('toggle_help') },
      },
    },
    number = false,
    relativenumber = false,
    signcolumn = 'yes',
  },
  trash = {
    cmd = 'trash',
    require_confirm = true,
  },
  actions = {
    change_dir = {
      global = false,
    },
    open_file = {
      quit_on_open = false,
      resize_window = true,
      window_picker = {
        enable = true,
        chars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890',
        exclude = {
          filetype = { 'notify', 'packer', 'qf', 'diff', 'fugitive', 'fugitiveblame' },
          buftype = { 'nofile', 'terminal', 'help' },
        },
      },
    },
  },
}
```

### Steps to reproduce

1.- Open files in vertical mode until the window picker appears.

### Expected behaviour

Window picker letters are displayed to choose the buffer where to open a new file

### Actual behaviour

The window picker is not displayed and after inserting the letter, for example, `A`, hitting enter key is required.

### PR DESCRIPTION
**Pull request by litao91**
closes #1361

### CODE REVIEW

**Comment by gegoune** [comment]
Doesn't that PR just hide the issue away?

**Comment by litao91** [comment]
@gegoune It's kind of hiding the problem. The problem is that when cmdheight is set to 0, the `print('pick window: ')` would hide the window indicator. This PR skip the printing thus the window indicator could be correctly displayed.

File: lua/nvim-tree/actions/open-file.lua [hunk 1]
```diff
@@ -99,7 +99,9 @@ local function pick_window()
   end
 
   vim.cmd "redraw"
-  print "Pick window: "
+  if vim.opt.cmdheight._value ~= 0 then
+      print "Pick window: "
```
Comment (kyazdani42) [inline lua/nvim-tree/actions/open-file.lua:103]: badly indented.

Comment (litao91) [reply lua/nvim-tree/actions/open-file.lua]: Fixed

**Review by kyazdani42 — APPROVED** [review]

**Pull request merged**